app/src/main/java/fr/free/nrw/commons/upload/mediaDetails/UploadMediaDetailFragment.java

```diff
@@ -180,6 +182,18 @@ private void init() {
             btnPrevious.setAlpha(1.0f);
         }
 
+        // If the image EXIF data contains the location, show the map icon with a red question mark
```

Comment:
I think the comments are mixed up?

Reply:
Ah silly mistake I should've give more attention to it.
Thanks for pointing it out.